documentation/northwind.adoc

```diff
@@ -1,151 +1,178 @@
-= Learn about graphs, how to load data, explore and query.
+= Learn graph fundamentals with Neo4j
 
-== What you will learn
+
+== Let's get started
 
 This guide will teach you about the power of graph databases. 
+We recommend it as the starting place for all new users as it will touch upon the concepts and tools you need to work with graphs. 
+
+=== What you will learn
+
+This guide will take around 10 minutes and by the end of it you will:
 
-It will take around 10 minutes and by the end of it you will:
+- Understand the *graph property model* and what problems it can help you solve.
+- Learn how to convert an understanding of a relational database model to a graph.
+- Import and model data from CSV files and map them to a graph.
+- Visually explore that data, without code, in a graph. 
+- Write your first simple graph queries using *Cypher*.
 
-Know what the graph property model is and what problems it can help you solve.
-Learn how to convert an understanding of a relational database model to a graph.
-Get familiar with Neo4j's tools to import, visualize, and query data in a graph.
+=== Let's get started! 
 
-The guide uses the classic (relational) Northwind dataset.
+We will be using the classic relational dataset known as Northwind. 
+It's time to bring this dataset and fictional company into an exciting graph-enabled future.
 
-1.  Model: What is a graph model and its elements
-2.  Load a graph from external CSV files
-3.  Explore the loaded data visually
-4.  Query patterns in your data
 
 == What is a graph?
 
-A graph database stores entities (nodes) and their relationships instead of tables, or documents. 
-Data is stored just like you might sketch ideas on a whiteboard. 
-Your data is managed without restricting it to a pre-defined schema, allowing a very flexible way of thinking and evolving it.
+=== Concepts
+
+A graph database stores *nodes* (`Customer`, `Product`, etc) and their *relationships* (e.g. `Customer PURCHASED Product`). 
+
+Unlike other types of database, which may use tables or documents, in a graph data is stored just like you might sketch ideas on a whiteboard.
 
 image::https://github.com/neo4j-graph-examples/northwind/raw/main/documentation/img/model.svg[]
 
-In relational databases, references to other rows and tables computed at query time through the use of joins.
-These can be complex, slow, and expensive.
+In relational databases, the relationships between rows and other tables are not stored. 
+Instead, you must write queries that use a `join`. These operations can be complex, slow, and expensive. 
```

Comment:
this bit (is actually also true for non-relational dbs), and saying they are not stored opens up a can of worms as people will state that "of course foreign keys are stored" ...
Question is what do we want to point out?
the need to add explicit join syntax to your queries (which will be alleviated by SQL-PG)
the issue that joins have to be computed each time a query runs?